extras/rya.merger/src/test/java/org/apache/rya/accumulo/mr/merge/RulesetCopyIT.java

```diff
@@ -144,32 +146,33 @@ private static RyaType literal(final String lit, final URI type) {
 
     @BeforeClass
     public static void setUpPerClass() throws Exception {
+        DemoUtilities.setupLogging(LoggingDetail.LIGHT);
         accumuloDualInstanceDriver = new AccumuloDualInstanceDriver(IS_MOCK, true, true, false, false);
         accumuloDualInstanceDriver.setUpInstances();
     }
 
     @Before
     public void setUpPerTest() throws Exception {
-        parentConfig = accumuloDualInstanceDriver.getParentConfig();
-        childConfig = accumuloDualInstanceDriver.getChildConfig();
-        accumuloDualInstanceDriver.setUpTables();
         accumuloDualInstanceDriver.setUpConfigs();
+        accumuloDualInstanceDriver.setUpTables();
         accumuloDualInstanceDriver.setUpDaos();
+        parentConfig = accumuloDualInstanceDriver.getParentConfig();
+        childConfig = accumuloDualInstanceDriver.getChildConfig();
         parentDao = accumuloDualInstanceDriver.getParentDao();
     }
 
     @After
     public void tearDownPerTest() throws Exception {
         log.info("tearDownPerTest(): tearing down now.");
-        accumuloDualInstanceDriver.tearDownDaos();
         accumuloDualInstanceDriver.tearDownTables();
+        accumuloDualInstanceDriver.tearDownDaos();
+        if (rulesetTool != null) {
+            rulesetTool.shutdown();
+        }
     }
 
     @AfterClass
     public static void tearDownPerClass() throws Exception {
-        if (rulesetTool != null) {
-            rulesetTool.shutdown();
-        }
         accumuloDualInstanceDriver.tearDown();
```

Comment:
After checking with everyone, I think we're now good with my changes since they teardown the tool's newly created child instance after each test like Andrew's changes.  My PR just keeps the same parent MAC instance up for the class and tears it down at the end of the class.